### ISSUE TITLE
cannot run tests if add custom task

### ISSUE THREAD

**Issue by ClaudioCarmeli** [glebm/i18n-tasks#230]
Can reproduce even with the `export_csv` task.
If you add the code as described and then run the rspec tests (`./spec/i18n_spec.rb`)
they fail on this line:

```
  include ::I18n::Tasks::Command::Collection
```

**Comment by glebm** [comment]
What is the error message and the backtrace? What does your config file look like?

**Comment by ClaudioCarmeli** [comment]
error and backtrace
```
 I18n does not have missing keys
     Failure/Error: include ::I18n::Tasks::Command::Collection

     NameError:
       uninitialized constant I18n::Tasks::Command
     # ./lib/i18n_csv_tasks.rb:4:in `<module:I18nCsvTasks>'
     # ./lib/i18n_csv_tasks.rb:3:in `<top (required)>'
     # (erubis:135:in `<main>'
     # /Users/claudiocarmeli/.rvm/gems/ruby-2.3.1@my-app/gems/erubis-2.7.0/lib/erubis/evaluator.rb:65:in `eval'
     # /Users/claudiocarmeli/.rvm/gems/ruby-2.3.1@my-app/gems/erubis-2.7.0/lib/erubis/evaluator.rb:65:in `result'
     # /Users/claudiocarmeli/.rvm/gems/ruby-2.3.1@my-app/gems/i18n-tasks-0.9.12/lib/i18n/tasks/configuration.rb:23:in `file_config'
     # /Users/claudiocarmeli/.rvm/gems/ruby-2.3.1@my-app/gems/i18n-tasks-0.9.12/lib/i18n/tasks/configuration.rb:37:in `config='
     # /Users/claudiocarmeli/.rvm/gems/ruby-2.3.1@my-app/gems/i18n-tasks-0.9.12/lib/i18n/tasks/base_task.rb:40:in `initialize'
     # ./spec/i18n_spec.rb:5:in `new'
     # ./spec/i18n_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./spec/i18n_spec.rb:6:in `block (2 levels) in <top (required)>'
     # ./spec/i18n_spec.rb:10:in `block (2 levels) in <top (required)>'
```

added the following to my config file:

```
## Custom CSV import and export tasks
# https://github.com/glebm/i18n-tasks/wiki/Custom-CSV-import-and-export-tasks
<% require './lib/i18n_csv_tasks' %>

csv:
  export:
    - ["faqs.*", "tmp/i18n-export/faqs.csv"]
    - "tmp/i18n-export/main.csv"
  import:
    - tmp/i18n-export/main.csv
    - tmp/i18n-export/faqs.csv
```

**Comment by glebm** [comment]
This is because the `Commands` module is not required when not running `i18n-tasks` from the CLI.

You need to add this to the custom task:

```ruby
require 'i18n/tasks/commands'
```

By the way, consider the https://github.com/halostatue/i18n-tasks-csv gem (it's the same as the code in the wiki but packaged as a gem). It doesn't do the `require` though.

I've updated the wiki.

**Comment by ClaudioCarmeli** [comment]
@glebm Thank you so much for getting back to me so quickly and with such helpful answers!
So should I use that gem instead? or would it cause the same issue? if I used the gem instead I wouldn't necessarily be able to go in and add the `require`. Am I right?

**Comment by halostatue** [comment]
I just released the updated gem (v1.1) with fixes suggested by @glebm.
